changelog/2236.removal

```diff
@@ -0,0 +1 @@
+- remove plugin preinit, we no longer need to do that because the namespace is initialized in the module now
```

Comment:
I suggest to make this more user oriented, the way it is now I doubt the general user will understand it. How about:

```
Remove internal ``_preloadplugins()`` function. This removal is part of the ``pytest_namespace()`` hook deprecation.
```